### PR TITLE
Add Up-to-Date Work

### PR DESCRIPTION
**Pull request by MatthewChastain**
This PR adds in all work on the DASSL Duck project up to the end of the Spring 2019 semester. The `DASSLduck.obj` file was added and overwritten with each successive version to keep the version control scheme intact. Images of past versions of the duck are in the images/ directory in specified folders for each past version. The README.md has also been updated to reflect the changes made to the repo.

### CODE REVIEW

**Review by afig — COMMENTED** [review]
Thanks for adding these files and images @MatthewChastain.

Really, the only suggestion I have is to perhaps provide the approximate size of the ducks created by the `.gcode` files. These sizes can be added somewhere in the readme and/or in the files names. If the sizes are in the readme, it might be sufficient to only give the size for the normal and XXXL sized ducks.

Due to the images and `.gcode` files, the repository is quite large, but I think it is okay since it won't be modified too frequently.

**Comment by smurthys** [comment]
Fantastic work @MatthewChastain. Thank you for the contribution. 

I confirm I am able to open the `obj` file in `Paint 3D` on Windows, but that is all I am able to verify at this point. I did make a couple of very minor edits to `README`, and agree with @afig's suggestions to add size information. Other than the size information, I feel these changes are ready to merge.

**The following suggestion/desire is for another PR:** It will be very helpful, if a "how to" document is added to provide a sequence of DIY steps to create a finished duck. This doc has two parts: the first part is includes the sequence of operations in various software tools to end up with a .obj file. The second part is the operations performed with a printed duck to produce a finished duck. I imagine especially the second part of the doc to take the form of DIY recipes commonly seen on the web where they show the result of each step so followers can verify result at each stage. More important, I think this doc will help the reader fully appreciate the amazing organization, effort, and discipline you have put into producing this artifact.

**Review by smurthys — APPROVED** [review]
Ready to merge 👍

**Comment by MatthewChastain** [comment]
I've added in the sizes of each resulting model from the .gcode files into the `README.md` but I don't think the table is rendering properly from the markdown, perhaps I'm missing something?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,35 @@
...
+Support Overhang Angle: 60 mm/s
+Build Plate Adhesion Type: Skirt
+```
+Support was used for the models in the pictures, however others have printed without support with no issues. 
+
+## Repository Content
+This repository contains the current version of the DASSL Duck in a .obj 3D file format which should be compatible for any slicing engine. For convenience there is also the `gcode/` directory that contains pre-compiled gcode for use on most standard FDM printers. There are multiple different sizes of the model with `DASSLduck.obj` being the standard size. The repository also contains an `images/` directory with pictures of the current version (v5) in the top-level, as well as images of previous versions in dedicated folders. There is no `v4` folder as there is almost no visual difference between `v4` and `v5` DASSL Ducks.
+
+## Gcode Sizes
+These are the sizes of the models that will be printed using the generated gcode files in the `gcode/` directory:
```
Comment (afig) [inline README.md]:
```suggestion
These are the sizes of the models that will be printed using the generated gcode files in the `gcode/` directory:

```

**Review by afig — COMMENTED** [review]
Thanks for adding the sizes @MatthewChastain. Looks like the issue with the tables is caused by there needing to be a blank line before the table.

**Review by afig — APPROVED** [review]
Good to merge after the suggested change has been accepted or otherwise addressed

**Pull request merged**